backend/src/test/java/com/redhat/cloud/notifications/routers/handlers/userconfig/UserConfigResourceTemplateModuleTest.java

```diff
@@ -0,0 +1,648 @@
+package com.redhat.cloud.notifications.routers.handlers.userconfig;
```

Comment:
Almost a copy of `UserConfigTest` with some adaptation accorting test cases, to test on app and event type without daily diest template, or event type without instant template at all.